src/_posts/platform/app/2000-01-01-review-apps.md

```diff
@@ -65,12 +66,26 @@ Child applications clone some information from the parent app:
 
 This default behavior can be customized using a `scalingo.json` file, see below as explained in the following section.
 
-It is however important to understand that the customization can allow to connect directly the Review App to the parent application database and addons, so Review Apps must be enabled only when trusted sources are allowed to create merge requests in the source code repositories. As a consequence, Scalingo prevents Review Apps from being created from forks of the original repository.
+It is however important to understand that the customization can allow to connect directly the Review App to the parent application database and addons, so Review Apps must be enabled only when trusted sources are allowed to create merge requests in the source code repositories. As a consequence, Scalingo prevents, by default, Review Apps from being created from forks of the original repository.
 
 {% note %}
 Databases content and other addons content won't be copied from the parent application to its child applications.
 {% endnote %}
 
+## Accepts Review Apps deployments from forks
+In case your project also accepts pull requests from forks, and you want to take advantage of the automatic deployment of the corresponding reviews apps, you can activate the automatic deployment of review apps from forks.
```

Comment:
**issue:** It is important to specify we are talking about forked repositories here

```suggestion
In case your project also accepts pull requests from forked repositories, and you want to take advantage of the automatic deployment of the corresponding reviews apps, you can activate the automatic deployment of review apps from forks.
```